### PR TITLE
Fix 575 usage ignore lookup

### PR DESCRIPTION
**Pull request by irina060981**
1. For the issue https://github.com/alpheios-project/components/issues/575
    Added data-ignore = "all"

2. Also I have fixed a bug with redefine wordUsageTab on a new word

3. Added some unit-tests - events, selection, word-list vue components

### CODE REVIEW

File: src/vue/components/word-list/word-language-panel.vue [hunk 1]
```diff
@@ -162,7 +162,9 @@ export default {
       this.showDeleteAllBox = true
     },
     async makeAllImportant () {
+      console.info('*****makeAllImportant start')
```
Comment (kirlat) [inline src/vue/components/word-list/word-language-panel.vue]: I think this needs to be removed

File: src/vue/components/word-list/word-language-panel.vue [hunk 2]
```diff
@@ -162,7 +162,9 @@ export default {
...
       await this.app.updateAllImportant(this.languageCode, true)
+      console.info('*****makeAllImportant after')
```
Comment (kirlat) [inline src/vue/components/word-list/word-language-panel.vue]: Same here

File: tests/lib/selection/media/html-selector.test.js_backup [hunk 3]
```diff
@@ -0,0 +1,160 @@
+/* eslint-env jest */
```
Comment (kirlat) [inline tests/lib/selection/media/html-selector.test.js_backup]: Do we need a backup file on the GitHub?

**Comment by kirlat** [comment]
Thanks for doing the tests!

File: src/vue/components/word-list/word-language-panel.vue (hunk 1 above)
Comment (irina060981) [reply src/vue/components/word-list/word-language-panel.vue]: Thanks for catching this!

File: tests/lib/selection/media/html-selector.test.js_backup (hunk 3 above)
Comment (irina060981) [reply tests/lib/selection/media/html-selector.test.js_backup]: Removed - missed this, thanks!

**Pull request merged**